scripts/build_demand_profiles.py

```diff
@@ -181,8 +181,19 @@ def build_demand_profiles(
     gegis_load = gegis_load.to_dataframe().reset_index().set_index("time")
     # filter load for analysed countries
     gegis_load = gegis_load.loc[gegis_load.region_code.isin(countries)]
-    logger.info(f"Load data scaled with scaling factor {scale}.")
-    gegis_load["Electricity demand"] *= scale
+
+    if scale is not None:
+        if isinstance(scale, dict):
```

Comment:
It may be useful to have a "DEFAULT" code, as in some cases, we may like only to describe a subset of users.
In the case it is a dict, I'd read a "DEFAULT" code (with .get("DEFAULT", 1.0)) and use this value for all countries that are not in the list.
What do you think?